modules/openapi-generator/src/main/resources/kotlin-client/build.gradle.mustache

```diff
@@ -46,5 +46,5 @@ dependencies {
     {{#threetenbp}}
     compile "org.threeten:threetenbp:1.4.0"
     {{/threetenbp}}
-    testCompile "io.kotlintest:kotlintest-runner-junit5:3.4.2"
+    testCompile "io.kotlintest:kotlintest-runner-junit5:3.1.0"
```

Comment:
@4brunu so downgrading the version actually fixes the issue...

Reply:
Yes